### PR TITLE
LAM-129 Refactored API documentation.

### PR DESCRIPTION
**Pull request by gouzouni625**
This pull request contains the refactored API documentation files to comply with the changes made on pull request #79 . The reason for creating a separate pull request is to avoid making #79 bigger than it already is.

### CODE REVIEW

**Comment by pkoro** [comment]
+1

File: webapp/api-doc/docs/ApplicationDelete.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+title: API | application delete
+description: Delete an uploaded application
+---
+
+# API - application delete - Description
+
+Application delete call, given an authentication oken through the header authentication,
```
Comment (themiszamani) [inline webapp/api-doc/docs/ApplicationDelete.md]: typo -- oken -> token

File: webapp/api-doc/docs/ApplicationUpload.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+---
+title: API | application upload
+description: Upload an application
+---
+
+# API - application upload - Description
+
+Application upload call, given an authentication oken through the header authentication,
```
Comment (themiszamani) [inline webapp/api-doc/docs/ApplicationUpload.md]: typo --> oken --> token

File: webapp/api-doc/docs/ApplicationUpload.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+will firstly check te validity of the token. If the token is invalid, the API wil reply with
```
Comment (themiszamani) [inline webapp/api-doc/docs/ApplicationUpload.md]: typo te -> the

**Pull request merged**